docs/source/conf.py

```diff
@@ -12,8 +12,6 @@
 #
 import os
 import sys
-sys.path.insert(0, os.path.abspath('../..'))
-sys.setrecursionlimit(1500)
```

Comment:
What was the aim of this originally? Just curious.

Reply:
it was so that Sphinx could import the package (added `../..` to the top of the python path). Now we are `pip install`ing the package, so it's not needed anymore.

Reply:
Gotcha. Neat.